### ISSUE TITLE
Present and fullScreen() not working on Processing 4.0b4(Apple Silicon)

### ISSUE THREAD

**Issue by jaegonlee** [benfry/processing4#372]
## Description

 
fullScreen() and present a full screen do not work on Processing 4b4.

## Current Behavior

Error said
```
A library used by this sketch relies on native code that is not available.
UnsatisfiedLinkError: /private/var/folders/zd/vgjzf6d54z17rllr0_s7wf280000gn/T/processing2184511611404286206menubar/libjAppleMenuBar.jnilib:
dlopen(/private/var/folders/zd/vgjzf6d54z17rllr0_s7wf280000gn/T/processing2184511611404286206menubar/libjAppleMenuBar.jnilib, 0x0001): 
tried: '/private/var/folders/zd/vgjzf6d54z17rllr0_s7wf280000gn/T/processing2184511611404286206menubar/libjAppleMenuBar.jnilib' 
(fat file, but missing compatible architecture (have 'x86_64,i386', need 'arm64e')), 
'/usr/lib/libjAppleMenuBar.jnilib' (no such file)

```

## Your Environment

 

* Processing version: Processing 4.0b4
* Operating System and OS version: macOS 12.1 / macbook pro 16 m1

## Possible Causes / Solutions

libjAppleMenuBar.jnilib needs to be compiled for apple silicon.
I tried build it but the API(Carbon, SetSystemUIMode) is too old.

**Comment by benfry** [comment]
Ok, good to know. We already have an issue open for that https://github.com/processing/processing4/issues/221, but this obviously increases the importance.

**Comment by benfry** [comment]
Fixed for 4.0 beta 5.
